ui/store/actions.test.js

```diff
@@ -91,15 +91,15 @@ describe('Actions', () => {
     originalNavigator = global.navigator;
 
     background = sinon.createStubInstance(MetaMaskController, {
-      getState: sinon.stub().callsFake((cb) => cb(null, [])),
+      getState: sinon.stub().resolves([]),
```

Comment:
just updating all the stubs and mocks to return promises/rejections. Also had to change `await` some `dispatch` calls, because all `background` calls now return asynchronously.